src/provider/states/running.rs

```diff
@@ -0,0 +1,91 @@
+use std::process::Child;
```

Comment:
I did not look at this file because I assume it'll change with systemd integration